index.html

```diff
@@ -1,55 +1,67 @@
 <!DOCTYPE html>    
 
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>MapComplete</title>
-    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css"
-          integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ=="
-          crossorigin=""/>
-    <link rel="stylesheet" href="./index.css"/>
-    <meta property="og:image" content="https://buurtnatuur.be/assets/BuurtnatuurFront.jpg" />
-    <meta property="og:type" content="website" />
-    <meta property="og:title" content="Buurtnatuur.be - samen natuur in kaart brengen" />
-    <meta property="og:description" content="Met deze tool kan je natuur in je buurt in kaart brengen en meer informatie geven over je favoriete plekje" />
-</head>
-<body>
-<div id="messagesboxmobilewrapper">
-    <div id="messagesboxmobile-scroll">
-    <div id="messagesboxmobile">    </div>
-    </div>
-    <div id="to-the-map">Loading... If this message persists, check if javascript is enabled and if no extension (uMatrix) is blocking it.</div>
-</div>
-
-<div id="topleft-tools">
-    <div id="userbadge-and-search">
-        <div id="userbadge">
-            Loading... If this message persists, check if javascript is enabled and if no extension (uMatrix) is
-            blocking it.
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>MapComplete</title>
+        <link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css"
+              integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ=="
+              crossorigin=""/>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
```

Comment:
Let's not use google fonts. Every time the page is loaded, the website pings to google that a user is visiting it. This is a small privacy issue

Reply:
Alternatively, it can be downloaded and be bundled with the website